### PR TITLE
coreos-livepxe-rootfs: add comment about separate `--fail`

### PR DESCRIPTION
**Pull request by jlebon**
Follow-up to 84905567 ("coreos-livepxe-rootfs: use `curl --fail`") and review feedback in #2201.

### CODE REVIEW

**Comment by jlebon** [comment]
(Killed CI for this. Not worth the resources.)

**Comment by bgilbert** [comment]
Mmm, I find the new text hard to parse (and it has a typo).  Could we clarify/expand a bit?

**Comment by jlebon** [comment]
Updated! Is that better?

File: overlay.d/05core/usr/lib/dracut/modules.d/35coreos-live/coreos-livepxe-rootfs.sh [hunk 1]
```diff
@@ -54,7 +54,10 @@ elif [[ -n "${rootfs_url}" ]]; then
 
     # Do a HEAD again but just once and with `--fail` so that if e.g. it's
     # missing, we get a clearer error than if it were part of the pipeline
-    # below.
+    # below (otherwise, the `curl` error emitted there would get lost among
```
Comment (bgilbert) [inline overlay.d/05core/usr/lib/dracut/modules.d/35coreos-live/coreos-livepxe-rootfs.sh]: Optional: parenthetical could be a full separate sentence instead.

**Review by bgilbert — DISMISSED** [review]
:laughing: Definitely expanded upon, yes.  Thanks.

**Review by bgilbert — APPROVED** [review]

**Review by dustymabe — APPROVED** [review]
LGTM
